### PR TITLE
Second branch

### PR DESCRIPTION
**Pull request by mistermaxis-zz**
The items are not positioned correctly as of yet and there is a lot of background colors.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @mistermaxis :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

* [x] Add a very descriptive README.
* [x] Please, do away with the background color in the Editor's Picks section.
* [x] Remove all the background pictures towards the bottom of your page.
* [x] Your page is missing the `Theory of Inflation` section.
* [x] Kindly complete your project before submitting for another review

### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Required changes:full_moon:
**Hi Max!
Great job so far on this project:clap::clap:. There are still some changes to make to get this project approved:muscle:**.

**Mandatory requirements**

- [x] Navigation bar

- It has a "Hamburger button", a search icon and a link with text on the leftmost side. Make the text a link.
- It has the NY Times image logo embedded in the center. Replace the text with the NY Times image logo.
- The NY Times logo is a link. Ensure it's a link
- 'Log in' elements on the rightmost side. Use a color picker to ensure you are picking the right color from the original page.

- [x] Main Content

- Work on the Theory of inflation. It's currently not close to the original.
- The footer section also currently doesn't look like the original. kindly work on it

- [x] Implementation details

- Elements aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). Good job on using float and flex in this project. You also need to use grid. Kindly refactor the codebase to account for this. 

**Take your time and work on the project to make it look very similar to the original. When you are done, submit it for another review.** 
Kindly drop your questions :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy Coding:blush:
Eric Enaburekhan.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

### Hello team 👋,

**Great job implementing the requested changes 👏.**

### Highlights

- **The linters are properly configured.**

### Mandatory Requirements.

1). **Create social media icons using CSS and HTML instead of using screenshots.**
2). **The theory of inflation looks unfinished and is overlapping with other sections of the page. See screenshot below.**

theoryInflation

3). **In the navbar, ensure the login and subscribe buttons are links.**

4). **The font color in the footer section should be dark. Also, the text should not be underlined.**

5). **Please use the [float](https://css-tricks.com/all-about-floats/) technique in your project as it is on the required techniques.**

6). **Remove the page horizontal scroll.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,690 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (krys2fa) [inline index.html]: - [x] [OPTIONAL] You may want to use a similar title since it is a clone.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @mistermaxis  👏 
You implemented most of the changes requested by the previous Code Reviewers. 💪 

### Status: Changes Required ♻️ 

**NB:** Review was done at a screen size of **1440px**. Kindly state your preferred screen size if your clone displays better on your machine.



- [x] [REQUIRED] The upper left part of the navigation bar on your clone should look like the original (see screenshots below for comparison). 

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2021-02-20 at 12 33 11 AM |  Screen Shot 2021-02-20 at 12 33 22 AM


- [x] [REQUIRED] You are missing an icon. The comparison below with the original demonstrates this.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2021-02-20 at 12 35 56 AM |  Screen Shot 2021-02-20 at 12 36 06 AM

- [x] [REQUIRED] You still have some overlapping content in the *Theory of Inflation* section as shown below:
Screen Shot 2021-02-20 at 12 38 50 AM


Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by mistermaxis-zz** [comment]
I made all the changes requested. I added the missing icon but it's a bit off. I'll keep working on it to make it look right. For the inflation section I tried to use percentages to make it better but I'm not sure it's going to look okay on different screens. Mine is 1920x1080 and looks as in the screenshot of the README file. Thanks to everybody who provided feedback for this project. You're the best!

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Hi, @mistermaxis Great job so far you made a great clone but you still have some minor changes that you need to take care of.

Please check the comments below.
:point_down: :point_down: :point_down: :point_down: :point_down: 

## Required Changes :red_circle::

- [x] Fix the link to your image, change uppercase letters to lowercase ones.
image

-  [x] Fix your texts with the images and make sure that they are not overlapping with the images:
image

- [x] Add some margin in the footer:
image

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**